src/liquid/components/ld-input/ld-input.tsx

```diff
@@ -161,6 +168,15 @@ export class LdInput implements InnerFocusable {
     }
   }
 
+  private handleKeyDown = (ev: KeyboardEvent) => {
+    if (
+      this.el.getAttribute('aria-disabled') === 'true' &&
+      !['ArrowLeft', 'ArrowRight', 'Tab'].includes(ev.code)
+    ) {
+      ev.preventDefault()
+    }
+  }
```

Comment:
I didn't find a way to focus the date picker indicator or open the date picker in the e2e tests. This is manually tested.